### DISCUSSION TITLE
How to alert if prometheus stopped sending metrics or prometheus had an outage

### DISCUSSION THREAD

**Discussion by tooptoop4** [grafana/grafana#45705] [Alerting]
Any template alert to catch when prometheus is down? as prometheus can't alert on itself

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
